<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forgetPassword</class>
 <widget class="QMainWindow" name="forgetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="userLine">
       <property name="styleSheet">
        <string notr="true">
background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;    border-style: outset;
    border-width: 2px;
    border-radius: 15px; 
    border-color: black;
    padding: 4px;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>UserName</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Security Key</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="SkeyLine">
       <property name="styleSheet">
        <string notr="true">
background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;    border-style: outset;
    border-width: 2px;
    border-radius: 15px; 
    border-color: black;
    padding: 4px;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>new Password</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="NPassLine">
       <property name="styleSheet">
        <string notr="true">
background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;    border-style: outset;
    border-width: 2px;
    border-radius: 15px; 
    border-color: black;
    padding: 4px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
background-color:rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;    border-style: outset;
    border-width: 2px;
    border-radius: 15px; 
    border-color: black;
    padding: 4px;</string>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 15px; 
    border-color: black;
    padding: 4px;color:rgb(0, 0, 0);
background-color:rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Back Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>821</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/photos/background.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="newRc.qrc">:/photos/background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="newRc.qrc"/>
 </resources>
 <connections/>
</ui>
